### PR TITLE
D2G: pass --init to docker run / podman run command

### LINKED ISSUE #7255: Command ABORTED on d2g worker
**Describe the bug**
As seen intermittently by our fuzzing instances, some instances fail with a `Command ABORTED` error.  See this [task](https://community-tc.services.mozilla.com/tasks/BXWx-CnNRhiaAP7kReXNVw/runs/0/logs/public/logs/live.log) for further details.

These tasks are running on generic worker with the payload converted by d2g.

### PR DESCRIPTION
**Pull request by petemoore**
Fixes #7255

### CODE REVIEW

**Comment by lotas** [comment]
Thanks for adding a quick fix!
Do you think it i will be possible to add an actual test for the `timeout` functionality? Since we didn't catch it before? 
Maybe we can test the actual time it takes to kill the container and compare if it is within some delta?

File: tools/d2g/d2gtest/testdata/testcases/artifacts_test.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ testSuite:
           - - bash
             - '-cx'
             - >-
-              timeout 3600 podman run -t --name taskcontainer
+              timeout 3600 podman run --init -t --name taskcontainer
```
Comment (lotas) [inline tools/d2g/d2gtest/testdata/testcases/artifacts_test.yml:36]: is there a test anywhere that is verifying the timeout part?

**Review by lotas — APPROVED** [review]

**Comment by petemoore** [comment]
> Thanks for adding a quick fix! Do you think it i will be possible to add an actual test for the `timeout` functionality? Since we didn't catch it before? Maybe we can test the actual time it takes to kill the container and compare if it is within some delta?

Good call. I've created #7273 for this. Thanks!

**Review by matt-boris — COMMENTED** [review]
This is great! 🎉

**Pull request merged**